lint_test/analysis_options.yaml

```
@@ -15,7 +15,7 @@ custom_lint:
       exclude:
         - class_name: Exclude
           method_name: avoidNumberOfParameters
-        - method_name: avoidNumberOfParameters          
+        - method_name: avoidNumberOfParameters               
```

Remove the extra spaces
```suggestion
        - method_name: avoidNumberOfParameters
```